src/graph/LookupExecutor.cpp

```diff
@@ -0,0 +1,499 @@
+                return;
+            }
+            resp_->set_rows(std::move(ret).value());
+        }
+    }
+    doFinish(Executor::ProcessControl::kNext);
+}
+
+void LookupExecutor::setOutputYields(SchemaWriter *outputSchema,
+                                     const std::shared_ptr<ResultSchemaProvider>& schema) {
+    if (schema == nullptr) {
+        return;
+    }
+    for (auto& col : returnCols_) {
+        outputSchema->appendCol(col, schema->getFieldType(col).type);
+    }
+}
+
+void LookupExecutor::processEdgeResult(EdgeRpcResponse &&result) {
+    auto all = result.responses();
+    std::shared_ptr<SchemaWriter> outputSchema;
+    std::unique_ptr<RowSetWriter> rsWriter;
+    std::shared_ptr<ResultSchemaProvider> schema = nullptr;
+    for (auto &resp : all) {
+        if (!resp.__isset.rows || resp.get_rows() == nullptr || resp.rows.empty()) {
+            continue;
+        }
+        if (outputSchema == nullptr) {
+            if (!yields_.empty()) {
+                schema = std::make_shared<ResultSchemaProvider>(*(resp.get_schema()));
+            }
+            outputSchema = std::make_shared<SchemaWriter>();
+            outputSchema->appendCol("edgeSrc", nebula::cpp2::SupportedType::VID);
+            outputSchema->appendCol("edgeDst", nebula::cpp2::SupportedType::VID);
+            outputSchema->appendCol("edgeRank", nebula::cpp2::SupportedType::INT);
+            setOutputYields(outputSchema.get(), schema);
+            rsWriter = std::make_unique<RowSetWriter>(outputSchema);
+        }
+        for (const auto& data : *resp.get_rows()) {
+            RowWriter writer(outputSchema);
+            const auto& edge = data.get_key();
+            writer << edge.get_src() << edge.get_dst() << edge.get_ranking();
+            for (auto& column : returnCols_) {
+                auto reader = RowReader::getRowReader(data.get_props(), schema);
+                auto res = RowReader::getPropByName(reader.get(), column);
+                if (!ok(res)) {
+                    LOG(ERROR) << "Skip the bad value for prop " << column;
+                    continue;
+                }
+                auto&& v = value(std::move(res));
+                auto status = Collector::collect(v, &writer);
+                if (!status.ok()) {
+                    LOG(ERROR) << "Collect prop error: " << status;
+                }
+            }
+            rsWriter->addRow(writer.encode());
+        }
+    }  // for `resp'
+    finishExecution(std::move(rsWriter));
+}
+
+void LookupExecutor::processVertexResult(VertexRpcResponse &&result) {
+    auto all = result.responses();
+    std::shared_ptr<SchemaWriter> outputSchema;
+    std::unique_ptr<RowSetWriter> rsWriter;
+    std::shared_ptr<ResultSchemaProvider> schema = nullptr;
+    for (auto &resp : all) {
+        if (!resp.__isset.rows || resp.get_rows() == nullptr || resp.rows.empty()) {
+            continue;
+        }
+        if (outputSchema == nullptr) {
```

Comment:
When no yield, the vertex didn't show in the output in console.
![image](https://user-images.githubusercontent.com/13706157/74010650-f3bcb300-49c0-11ea-8a4b-91a8b25f5126.png)

Reply:
Fiexed